tpls/appcache-frame.tpl

```diff
@@ -90,9 +90,6 @@
         clearInterval(downloadingInterval);
         downloadingInterval = null;
       }
-
-      applicationCache.removeEventListener('downloading', onDownloadingEvent);
-      applicationCache.removeEventListener('progress', onDownloadingEvent);
```

Comment:
Okay, problem with removing this is that `progress` happens multiple times, i.e. it may happen once per file. So removing this cleanup makes `onUpdating` event fire very often.